src/backend/api/handlers/event.py

```diff
@@ -147,6 +148,19 @@ def event_teams_statuses(event_key: EventKey) -> Response:
     return profiled_jsonify(statuses)
 
 
+@api_authenticated
+@validate_keys
+@cached_public
+def event_teams_media(event_key: EventKey) -> Response:
+    track_call_after_response("event/teams/media", event_key)
+
+    query = EventTeamsPreferredMediasQuery(event_key=event_key).fetch_dict(
```

Comment:
this only gives the preferred media, aka the 1-2 good robot pics for the team. Is that the intent?

Reply:
Yeah thats the intent.

Reply:
Should the endpoint be renamed/documented to reflect this? Why not just make this use all the media, since `preferred` is an available property anyways?

Reply:
I can make it all.